src/System.Runtime.Serialization.Json/tests/Performance/DcjsDeserializationTests.cs

```diff
@@ -0,0 +1,141 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Xunit.Performance;
+using SerializationTypes;
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.Json;
+using System.Xml;
+
+namespace System.Runtime.Tests.Performance
+{
+    public class DcjsDeserializationTests
+    {
+        public void RunDcjsDeserializeTest(object obj, int iterations)
```

Comment:
The class name uses "Deserialization" while this method name uses "Deserialize". Do we want to keep them consistent?